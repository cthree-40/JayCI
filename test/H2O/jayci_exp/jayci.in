&general
electrons=10
orbitals=24
nfrozen=1
ndocc=4
nactive=0
xlevel=2
/
&dalginfo
maxiter=25
krymax = 8
krymin = 3
restol = 1d-5
nroots = 1
prediagr = 1
iguessdim = 4
/
