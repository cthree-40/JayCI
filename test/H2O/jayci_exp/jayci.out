 ------------------------------------
 jayci.x 
 ------------------------------------
 
 &general input values:
  electrons =   10
  orbitals  =   24
  nfrozen   =    1
  ndocc     =    4
  nactive   =    0
  xlevel    =    2
 
 
 &expansion input values:
  ci_electrons =               8
  ci_orbitals  =              23
  alpha strings=            1103
  beta  strings=            1103
  determinants =            7981
 
 
 &dalginfo input values:
  maxiter      =              20
  krymin       =               3
  krymax       =               8
  nroots       =               1
  prediagr     =               1
  iguessdim    =               4
  restol       =      0.00001000
  refdim       =              50
 

 Nuclear repulsion energy:     9.16048714
 Frozen-core contribution:   -61.30695281
 Total contribution:         -52.14646567

 Hartree Fock Energy:    -76.02650823
 Final CI energy =    -76.23008839
